UnityProject/Assets/Scripts/Health/Electrocution.cs

```diff
@@ -1,162 +1,50 @@
-using System;
 using UnityEngine;
 
 /// <summary>
-/// For electrocuting a victim. NPCs not emplemented, just (human) players.
+/// The rated severity of an electrocution.
 /// </summary>
-public class Electrocution
+public enum ElectrocutionSeverity
 {
-	/// <summary>
-	/// The electrocution severity.
-	/// <para>None: no potential difference (no side effects)</para> 
-	/// <para>Mild: exposure to voltage like that of a small battery (no consequential effects)</para>
-	/// <para>Painful: imagine an electric fence (drop actively held item, minor burns)</para>
-	/// <para>Lethal: high voltage line (stun, severe burns, death possible)</para>
-	/// </summary>
-	public enum Severity
-	{
-		None,
-		Mild,
-		Painful,
-		Lethal
-	}
+	/// <summary> No potential difference (no side effects) </summary>
```

Comment:
Severity seems very specific to humans, why is it an enum and not just part of the internal implementation in PlayerHealth / LivingHealthBehavior? Then those components can decide how they want to respond based on the strength of the electrocution. I don't think this should be part of Electrocution,, each implementation should be able to set the range of their behavior.